tests/misc_env/sosreport.py

```diff
@@ -10,21 +10,25 @@
 
 def run(**kw):
     """
+    Entry point for execution used by cephci framework.
```

Comment:
I would suggest to update doc strings as per google doc style.

Reply:
Done.